syntax = "proto3";

message Item {
    string key = 1;
    uint32 value = 2;
}

message TimeProfilerInfo {
    repeated Item item = 1;
}

message BatchHashMessage {
    repeated uint64 batch_hash = 1;
    uint32 merged_batch_size = 2;
    uint32 total_batch_size = 3;
    uint32 concurrency_num = 4;
    repeated string server_labels = 5;
}